nservicebus/upgrades/support-policy.md

```diff
@@ -9,7 +9,7 @@ related:
 
 ## Supported Versions
 
-Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported, or any minor version released within the last year.
+Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported. For the NServiceBus package only (i.e. excluding downstream packages such as [transports](/nservicebus/transports/) and [persistence](/nservicebus/persistence/)), any minor version released within the last year is supported.
```

Comment:
@indualagarsamy I like the idea of actually having separate sections like you've suggested, but the wording around minor versions is incorrect. The first paragraph is saying that all minors are supported in the 3 year period, which is incorrect.

Here's my suggested revision:

>NServiceBus:
Particular Software supports all major versions released within the last 3 years, For each supported major version, the latest minor version is supported. In addition, any minor versions released in the past year are also supported.

>Downstream packages:
For downstream packages that use NServiceBus, for example, all the transports, persistences, containers, etc., Particular Software supports all major versions released within the last 3 years. For each supported major version, only the latest minor version is supported.

I also think we should add something to make it explicit about which patch versions are supported. Something like 

> NOTE: For both cases, for each supported minor version, only the latest patch version is supported.